module.go

```diff
@@ -14,5 +15,6 @@ var Module = fx.Options(
 	logrus.Module,
 	metrics.Module,
 	health.Module,
+	readyness.Module,
```

Comment:
I would load it semantically before health, since the services has to be ready for health to take action